rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -164,13 +124,13 @@ def __next__(self):
             elif self.tensor_dtype == types.FLOAT16:
                 return self.out.astype(np.float16), self.res, self.l, self.num_bboxes_arr
         elif (self.loader._name == "TFRecordReaderClassification"):
-            if(self.loader._oneHotEncoding == True):
-                self.labels = np.zeros((self.bs)*(self.loader._numOfClasses),dtype = "int32")
-                self.loader.GetOneHotEncodedLabels(self.labels, device="cpu")
-                self.labels = np.reshape(self.labels, (-1, self.bs, self.loader._numOfClasses))
+            if(self.loader._one_hot_encoding == True):
```

Comment:
max_rows has been declared twice with same value in the above and looks like its hardcoded instead of calculating. 
Has this been checked?